content/builds/titan/hammer_of_the_wormgod.md

```diff
@@ -23,15 +23,14 @@ Discipline : 50+
 Intellect  : -
 Strength   : 50+
 ```
-* A "`-`" means the stat isn't a priority for the build.
 
-## Exotic Gear
-`Armor` - [Wormgod Caress](https://www.light.gg/db/items/1734844651/wormgod-caress/) will give you the highest possible damage output witha stacking buff. [Synthoceps](https://www.light.gg/db/items/241462142/synthoceps/) are not as powerful but provide a very consistent buff that's easier to manage.
+### Exotic Gear
```

Comment:
Nitpick: I thought you wanted to change up how this layout worked to focus on Armor here and move the weapon down to the weapons section.  It reads fine as is though.

Reply:
looks like i pushed an earlier version lol, should be fixed now